syntax = "proto3";

message Message {
    oneof msg {
        string text_message = 1;
        string acknowledgment = 2;
    }
    int64 sender_id = 3;
    int64 destination_id = 4; 
    int64 sender_port = 5;
}
